deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+
+# export KUBE_NAMESPACE=${ENVIRONMENT:-$DRONE_DEPLOY_TO}
+export KUBE_NAMESPACE="hocs-delta"
+# export KUBE_TOKEN=${KUBE_TOKEN}
```

Comment:
Uncomment this: although it doesn't actually do much, the `set -u` further up will ensure the script crashes out here if `KUBE_TOKEN` is undefined. Without this line it'll crash at the bottom when `kd` tries to consume it.